examples/scripts/ct_symcone_tv_padmm.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+TV-Regularized Cone Beam CT for Symmetric Objects
+=================================================
+
+This example demonstrates a total variation (TV) regularized
+reconstruction for cone beam CT of a cylindrically symmetric object,
+by solving the problem
+
+  $$\mathrm{argmin}_{\mathbf{x}} \; (1/2) \| \mathbf{y} - C \mathbf{x}
+  \|_2^2 + \lambda \| D \mathbf{x} \|_1 \;,$$
+
+where $C$ is a single-view X-ray transform (with an implementation based
+on a projector from the AXITOM package :cite:`olufsen-2019-axitom`),
+$\mathbf{y}$ is the measured data, $D$ is a 2D finite difference
+operator, and $\mathbf{x}$ is the solution.
+"""
+
+
+import numpy as np
+
+import scico.numpy as snp
+from scico import functional, linop, loss, metric, plot
+from scico.examples import create_circular_phantom
+from scico.linop.xray.symcone import SymConeXRayTransform
+from scico.optimize import ProximalADMM
+from scico.util import device_info
+
+"""
+Create a ground truth image.
+"""
+N = 256  # image size
+x_gt = create_circular_phantom((N, N), [0.4 * N, 0.2 * N, 0.1 * N], [1, 0, 0.5])
+
+
+"""
+Set up the forward operator and create a test measurement.
+"""
+C = SymConeXRayTransform(x_gt.shape, 5e2 * N, 6e2 * N, num_slabs=4)
```

Comment:
It would be more clear to use named arguments for the `obj_dist` and `det_dist`.